<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageMatcher</class>
 <widget class="QDialog" name="ImageMatcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="imgPath1"/>
      </item>
      <item>
       <widget class="QPushButton" name="imgButton1">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Image 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="imgPath2"/>
      </item>
      <item>
       <widget class="QPushButton" name="imgButton2">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Key 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="keyPath1"/>
      </item>
      <item>
       <widget class="QPushButton" name="keyButton1">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Key 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="keyPath2"/>
      </item>
      <item>
       <widget class="QPushButton" name="keyButton2">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
